src/workloads/execution/ExpressiveQueries.yml

```diff
@@ -0,0 +1,207 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+
+Actors:
+
+# Phases:
+# 1. Insert data
+# 2. Fsync
+# 3. Plain query (without index or projection)
+# 4. Query with projection, but without index
+# 5. Create index
+# 6. Fsync
+# 7. Query with projection and with index
+# 8. Drop collection
+
+- Name: InsertData
+  Type: Loader
+  Threads: 1 # Loader doesn't support multiple threads for a single collection
```

Comment:
FYI: This has come up recently and led to https://jira.mongodb.org/browse/TIG-2938. Consider adding a comment and/or voting for the ticket. Every bit of evidence helps when we all try to schedule and prioritize these things.